pyproject.toml

```
@@ -36,6 +36,9 @@ dependencies = [
 [tool.setuptools]
 platforms = ["any"]
 
+[tool.setuptools.packages]
+find = { where = ["deepinv"], exclude = ["results"] }
```

I think you can keep the `where` directive as we only aim to install this one and it will avoid maintaining an exclude list for all possible subfolders.
```suggestion
find = { where = ["deepinv"] }
```